Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -677,7 +677,16 @@ float PlayerBase::SpaceObjDamaged(uint space_obj, uint attacking_space_obj, floa
 			{
 				if (set_plugin_debug > 1)
 					ConPrint(L"PlayerBase::damaged space_obj=%u\n", space_obj);
-				hostile_tags[charname] = charname;
+				
+				if (hostile_tags_damage.find(charname) != hostile_tags_damage.end())
+				{
+					if (hostile_tags_damage[charname] < damage_threshold)
+						hostile_tags_damage[charname] += damage;
+					else hostile_tags[charname] = charname;
+				}
+				else hostile_tags_damage[charname] = damage;
```

Comment:
this doesn't account for the first shot being over the threshold?

Reply:
It accounts now